tests/x64_context_test.py

```diff
@@ -81,12 +69,12 @@ def test_correctly_capture_default(self, jit, enable_or_disable):
 
   @unittest.skipIf(jtu.device_under_test() != "cpu", "Test presumes CPU precision")
   @parameterized.named_parameters(jtu.cases_from_list(
-      {"testcase_name": "_jit={}".format(jit), "jit": jit}
-      for jit in ["python", "cpp", None]))
+      {"testcase_name": f"_jit={jit._name}", "jit": jit}
+      for jit in jtu.JIT_IMPLEMENTATION))
   def test_near_singular_inverse(self, jit):
     rng = jtu.rand_default(self.rng())
 
-    @partial(_maybe_jit, jit, static_argnums=1)
+    @partial(jit, static_argnums=1)
```

Comment:
This test case fails because static_argnums isn't a valid argument to `lambda x:x`.

Reply:
Fixed